src/status_im/ui/components/list/views.cljs

```diff
@@ -82,6 +82,12 @@
   [react/view {:style (merge style styles/item-checkbox)}
    [checkbox/checkbox props]])
 
+(defn list-item-with-checkbox [{:keys [on-value-change checked? plain-checkbox?] :as props} item]
+  (let [handler  #(on-value-change (not checked?))
```

Comment:
Can you ellaborate about `plain-checkbox?` ? Might be worth some doc.

Reply:
@jeluard sorry, I think `plain-checkbox?` sounds a little bit ambiguous. What it does is just avoid the checkboxes to be wrapped with `item-checkbox` view component that has some flexbox styles applied. We don't need it for contact list view checkboxes. Perhaps better naming for it would be `no-style?`.

Reply:
Sounds good as is.